index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
```

Comment:
```suggestion
<h1> Hello World </h1>
```
I think this is cool